### DISCUSSION TITLE
Welcome to ME2120 - Fall 2024!

### DISCUSSION THREAD

**Discussion by cooperrc** [cooperrc/engineering-dynamics#13] [General]
Greetings!

Welcome to our Applied Mechanics II: Engineering Dynamics (ME2120 section 001) course discussion board! Post questions, suggestions, and discoveries here for everyone to benefit from our learning community discussions. 

To kick off our discussions, 

1. What is a job posting that would require skills from this class?
2. What do you miss about Summer?
3. Show us an equation in $\LaTeX$ using `$\LaTeX$` and describe why you would use it and what it does

**Comment by jn985** [comment]
1. A job posting that would require skills from this class would be a sports product designer. Designing precision athletic equipment requires the ability to prototype, as well as the ability to perform dynamics calculations to determine how the gear will perform under stress.

2. I miss being able to travel during the summer.

3. $a^2 + b^2 = c^2$ 
The Pythagorean theorem is definitely one of the simpler formulas, but is still very powerful in statics and dynamics when accounting for the magnitudes of various forces acting on an object. It allows us to determine an unknown side length of a right triangle when we know the other two sides.

**Comment by MattBarber7** [comment]
1. A job that requires this class is bridge building and design. Large spanning bridges sway slightly in the wind, and while bridges should not move, the wind loads cause them to sway. This movement causes a dynamic moment onto the supports of the bridge, and this dynamic moment will be learned about in this class.
2. I miss going on cool adventures with my friends and family.
3. $\ F=ma$
This equation is used all of the time in physics. It describes the mass and acceleration of an object, and how it relates to its force. This acceleration can be dynamic as we will see in this class, and we can have 0 acceleration, meaning an object is not in motion. I would use this to determine what forces are being created from dynamic movements.

**Comment by MadelynOShaughnessy** [comment]
1. A job that requires this class may be the design of prosthetics to ensure the product can withstand the force the body may exert on it. A prosthetic especially one for the lower limbs must support the body's weight which can be calculated using methods learned in this class. In addition, power is important if someone needs a prosthetic for athletics. 
2. I miss being able to relax and my coworkers from my job at home. 
3. $\ P=W/t$
This equation represents power which equals work over the change in time. This equation may be used in this class because it relates to power, energy, and momentum.

**Comment by cooperrc** [comment]
1. job post _coming soon_
2. I miss sailing in the Summer
3. $\sigma_{ij} = C_{ijkl}\epsilon_{kl}$ This is Hooke's law in its full form, saying that stress is always proportional to strain

$\vec{F} = m \vec{a}$

**Comment by AchilleDeTroie** [comment]
1./ For an aerodynamicist job
2./ The south of france 
3./ $\exp^{i \pi} + 1 = 0$

**Comment by marceldabek** [comment]
1. An Automotive Engineer needs a strong background in dynamics because they have to understand how forces and motions interact within a vehicle's systems. Suspension, steering, and braking systems all are important in the dynamics realm.
2. Seeing most of my family at get togethers
3. $F = ma$
Newtons second law describes the relationship between an object's mass, its acceleration, and the resulting force. In this class I assume we will often use it because it is fundamental.

**Comment by moerakamimura** [comment]
1. A job posting that might require this is a civil engineer to be able to see how different forces interact with each other when building a suspension bridge for instance. 
2. I miss the food and seeing family and friends back home.
3. F = ma

**Comment by bensafran42** [comment]
1. As a mechanical engineer working at a submarine manufacturing place you would need dynamics to calculate the physics that occurring on the moving parts of the submarine. 
2. Going out my boat with my friends.
3. $\pi r^2\$
The area of a circle is very useful for force calculations as it used a lot in sheer and stress calculations.

**Comment by sway2704** [comment]
1.  A job listing such as a physicist would require skill from this class and the background of kinematics and how objects in motion interact with one another. 
2. Being in Puerto Rico
3. E =mc^2 , which is Einsteins mass-energy equivalence equation.

**Comment by michaelsbriscoe** [comment]
1. A job posting that would require skills from this class would be a Aerospace Engineering job designing planes. Dynamics and physics are very important for the function and movement of airplanes and skills from this class would be required.
2. I miss having my room to myself at home during the summer.
3.  $\[ V = IR\]$ This is Ohms law and it describes the relationship between current, resistance and voltage within circuits. You can use it to find and calculate different measurements of the variables within a circuit.

**Comment by cursingcookie** [comment]
1. Mechatronics and robotics engineers need a strong understanding of dynamics in to ensure their machines work as precisely  as possible, this is especially true for vehicles containing robotics.
2. I miss the beach
3. $\oint_S \mathbf{E} \cdot d\mathbf{A} = \frac{Q_{\text{enc}}}{\varepsilon_0}$ Gauss' law

**Comment by alexdeleva** [comment]
1. Robotics design
2. Being with family & hometown friends
3. $f^2 = c \left(\frac{E \cdot I}{l^4 \cdot p \cdot A}\right)$ this is variance for probability

**Comment by k593** [comment]
1. An aeronautical engineer would require an understanding of dynamics. As they design machines that move in certain ways due to interactions with outside elements such as wind and pressure in the atmosphere.
2. Going to the beach.
3. $PV = nRT$ is the ideal gas law. You would use it when calculating volume, pressure, or temperature of a gas in a problem when given certain known variables.

**Comment by NiFoley** [comment]
1) a drafter would need the knowledge of dynamics from this class so that the designs can correctly accommodate the application.

2) I miss riding my sport bike everyday
3 [
V
=
I
R
]
 This is Ohms law and it describes the relationship between current, resistance and voltage within circuits. You can use it to find and calculate different measurements of the variables within a circuit.

**Comment by nicholasmorenski** [comment]
1. Understanding properties and behaviors of materials under stress is imperative for all mechanical engineers to know. Ideally that skill would be highlighted in job listing for the time of work I'd like to do.
2. I miss fishing on the weekend
3. $\sigma_b = \frac{F}{A}$

**Comment by jasonsigler** [comment]
1. A job that requires this class would be a person who designs engines because they have to understand the forces that engines withstand.
2. Having ac in my house
3. $A = \frac{1}{2} b \times h$  this is the formula for the area of a triangle

**Comment by rebernas** [comment]
1.  A job posting for a biomechanics research scientist working on modeling human motion would require the skills learned in this class. They would need to understand how the different joints in the body move, which would require background in dynamics.
2. I miss the people that I worked with at the REU I did this summer in Washington. 
3. $s = ut + \frac{1}{2}at^2$ is a kinematics equation that allows you to predict the position of an object based on its acceleration and velocity.

**Comment by Michaelben2004** [comment]
1.  Mechanical engineers will need skills from this class because they must know dynamics to work with mechanical parts like the velocity of something 
2. I miss waking up late 
3. e^{i\pi} + 1 = 0 
4. Eulers equation to solve complex fluid problems

**Comment by ChristinaPortera** [comment]
1. A job in aerospace engineering would require a strong background in dynamics because they must design and analyze the behavior of spacecrafts. 
2. I miss going to the beach during the summer
3. $c^2 = a^2 + b^2$ is the Pythagorean Theorem and it can be used to find magnitudes of vectors and unknown angles or sides of triangles.

**Comment by lruch24** [comment]
1.  An aerospace job designing aspects of a rocket would need dynamics for various areas of the rocket.
2.  I miss the free time to be able to play guitar.
3.  $\rho = m/v$ is the equation for density which could be used when comparing different materials for a project.

**Comment by robk14** [comment]
1. A job that would need skills from this class would be a manufacturing robotics employee. They would need to know the forces acting on the robots so that they do not break and can handle changing force loads. 
2. I miss having a few days off to go on a trip and spend time with my friends. 
3. $A = \pi r^2$ : you can use this equation to find the area of a circle which could be the cross sectional area of a column or cylinder.

**Comment by jackbrown14** [comment]
1. Biomechanics would require skills from this class working with devices such as prosthetics to scanning machines.
2. Seeing my home friends.
3. $x = x_0 + v_0 t + \frac{1}{2} a t^2$  This is the kinematic equation involving position.

**Comment by khansamiii** [comment]
1. An exampe of a  job posting that would require skills from this class would be a car manufacturer. For many systems within a car, such as steering, wheel and axle requires movement of multiple components along with forces attached with each component. 

2. Something I miss from this summer was not having to go to class, and being able to do anything stress free. Just chillin.

3.  $E = mc^2$ This is Einsteins famous equation.

**Comment by emilyrussell14** [comment]
1. A job posting that would require skills from this class would be bridge design. Since we are in a moving world where the forces acting on an object such as a bridge are not always 0, they would need the skills for this class to understand computational geometry and forces. 
2. I miss the freedom of summer in being able to do whatever I wanted whenever I want
3. $\ F = ma$ is the equation I will use for the sum of forces on an object

**Comment by fionachang04** [comment]
1.  A job posting that requires skills from this class could include an engineer in the automotive industry, who will need to deal with moving vehicles and dynamic forces.
2.  I miss being able to go to the beach all the time
3.  $\[P = V \times I\]$ represents the power formula, used to calculate watts

**Comment by JuliaMiller234** [comment]
1. An aeronautics engineer would need skills from this class to create safe and useful planes and turbines. Dynamics is extremely useful in this job and improper knowledge of dynamics could result in deaths and crashes.
2. I miss seeing my family and dog everyday and not having lots of responsibities.
3. \mathcal{E} = -\frac{d\Phi_B}{dt}
This is the equation for Faraday's Law. Faraday's law states that when a magnetic field is put into motion through a wire loop, it creates a voltage that tries to oppose the magnetic field's movement.

**Comment by NathanGrzybowski** [comment]
1.) A job posting that might require this class is the building of submarines, because during the building you need to know the forces that will be acting on the sub and how to combat them.
2.) fishing
3.) /(P=MV)/  this is momentum which is important for measuring the motion of a moving body.

**Comment by connecticutuser** [comment]
1. A dynamics engineer would have to be an expert on forces and how they interact with each other. 
2. Traveling and having free time. 
3. $PV = nRT$, the ideal gas law. An equation describing the relationship between temperature, pressure, and volume for gases.

**Comment by Abram133** [comment]
1. A job posting that might require applied mechanics would be a Lego Set Designer. You would have to figure out the most effective way to produce a set while taking in account stress, force, possible deformation.
2. I miss having a stable income in the summer
3. My favorite equation is $\[ \vec{F} = 0 \]$ because I enjoy calculating for statics

**Comment by C-vonk** [comment]
1. A mechanical Engineer working in modeling aircraft movement.
2. racing on my brothers racing sim
3. $\rho = m / v$ this is density equation and it is used in all sorts of settings such as bouyency

**Comment by Ibrahim-shub** [comment]
1) Building and manufacturing oil tankers, because buoyancy is a force.
2) I miss styling hair for all my customers.
3) F = ma, the equation for Newtons second law of motion; The relationship between force, mass, and acceleration.

**Comment by chr979** [comment]
1. A job involving robotic design would use skills from this class because they need to be able to design parts and their movement precisely.
2. I miss going to the beach in the summer. 
3. $v=v_0 + at$ This is a kinematic equation used to describe linear motion with a constant acceleration.

**Comment by kendra-baker** [comment]
1. A job posting that can require this is a robotics engineer.
2. Something I miss about summer is spending time with my family, friends, and my dog.
3. $pV = nRT\$. This is the ideal gas law which signifies the relationship between pressure, volume, the number of moles, and temperature of the gas, as well as the universal gas constant, R.

**Comment by YukiAskew** [comment]
1. Such as surgent technology you would need to have dynamics skills because you are designing the robot's motion to be fine and precise and that comes down to the skills we will learn in this class.
2. I miss fishing and golfing
3. $\vec F = m \vec a$
     This is Newton's Second Law and means more mass an object has, the more force you need to accelerate it.

**Comment by Edwinxg** [comment]
1. A job posting that would require skills from this class would be any aerospace engineering job. Engineering dynamics are essential for multiple things in this field such as testing the stability, performance and control of various aircraft and spacecraft.
2. I miss being in Puerto Rico
3. F=ma this is the mathematical equation that represents Newton's second law and is used very often in physics. I can assume it will be used a lot in this class due to the fact its fundamental to most STEM courses

**Comment by alondral99** [comment]
1. A job posting that would require skills from this class would be a mechanical engineering job. Since mechanical engineers design and develop moving technology, understanding dynamics become a crucial skill to these engineers.
2. I miss taking day trips with my friends out of town.
3. $s = ut +\frac{1}{2}at^2$ is the kinematic equation and it is my favorite equation because it is used to calculate the displacement of any object.

**Comment by CaptainlegendHM** [comment]
1. Mechanical / automatic design
2. Not worrying about homework and grades
3. $W = \frac{1}{2}* m * (v^2)$, this is the work-energy equation, it helps us calculate how much work we need to get certain kinematic energy or speed

**Comment by EmilyDAiuto** [comment]
1. A patent attorney needs a strong background in dynamics because they are required to understand how systems work within objects to make sure their clients are getting the correct patents to protect their assets.
2. I miss seeing my cousins often and my dog every day.
3. $F = \frac{G m_1 m_2}{r^2}$, this is the universal gravitation equation, it allows us to figure out the gravitational force between two objects by using their masses, the distance between them, and the gravitational constant.

**Comment by elena-hartley** [comment]
1. Propulsion motor engineers would need extensive knowledge of statics and dynamics in order to safely and confidently design parts and ensure the parts could withstand use and work well together. 
2. I miss hiking with my dog 
3. $pV=nRT$ is the ideal gas law and it allows us to solve for different properties of an ideal gas. When some of the properties are known we can solve for the remaining.

**Comment by joshuajimenez1** [comment]
1. A robotics job would require dynamics skills to ensure good design for functioning systems and movement of machines
2. I miss snorkeling in the summer
3. $F = ma$ Newton's second law which is universal and used for many physics applications

**Comment by plalsd** [comment]
1. A Civil Engineering position would require skills learned in a dynamics class to deal with unexpected events such as earthquakes or other sudden loads. 
2. I miss having free time during the summer to do whatever I want.
3.  $\beta = \frac{Qc}{Wcycle} = \frac{Qc}{Qh-Qc}$ This is an important equation in thermodynamics and helps describe the efficiency of a refrigeration cycle. In mechanical engineering, there are often times when you want to cool a system to prevent damage.

**Comment by ACarr576** [comment]
1. An aviation designer would need a very in depth understanding in dynamics, because of the forces that interact with an airplane ahs everything to do with dynamics.
2. I miss going on my hikes and backpacking trips with my friends and family this summer.
3.  KE = \frac{1}{2}mv^{2}  This is the kinetic energy equation, this would be used in physics when trying to discover the kinetic energy when mass and velocity are known.

**Comment by ejk21002** [comment]
1. A job posting that would require skills from this class could be in biomedical devices and equipment in order to make sure they are safe and secure to use for patients.
2. I miss relaxing and being with my dog
3. ${v} = \mathbf{v}_0 + \mathbf{a} t$. This equation is used for motion under constant acceleration.

**Comment by cdumouchel36** [comment]
1. One example of a job posting requiring dynamics knowledge is an aerospace engineer, because the design of certain satellites, spacecraft, etc. must take into account the way forces are acting on the object as it moves.
2. I miss seeing my best friend every day.
3. $a^2+b^2=c^2$ The Pythagorean theorem which is used to calculate side lengths of right triangles

**Comment by steve-O613** [comment]
1. automobile engineer
2. fishing, swimming, drinking, flip flops
3. $F = m \cdot a$ this is newtons second law of motion and it relates how objects accelerate given their mass and applied force.

**Comment by EvanBliven** [comment]
1. Jobs around automotive design and engineering would require skills in dynamics
2. I miss spending time with friends from out of state
3. $\Delta K = \frac{1}{2} m v_f^2 - \frac{1}{2} m v_i^2$  This is the kinetic energy equation, used to find the kinetic energy of an object in motion

**Comment by erick220** [comment]
1.  An job example that requires skills from dynamics is a mechanical engineer
2.  I miss not having homework and classes to wake up early to
3. $A$ = $b$ * $h$ is the equation to solve for area by multiplying base times height

**Comment by oliver-janssen** [comment]
1. Skills from this class would be required in aerospace engineering
2. I miss seeing my home friends and not having homework
3. $x = x_{0} + v_{0}*t + (1/2)at^2$ This equation is used to calculate the position of an object based on its initial position, initial velocity, acceleration, and time

**Comment by Nick-Grat** [comment]
1. A job in aerospace revolving around planes and how a moving plane has to deal with drag and what can be done to minimize that. 
2. Not being able to make money 
3. \( f = ma \) and it is used to calculate the force of an object with acceleration and mass are given

**Comment by MTyburski-LBX** [comment]
1. A job in the automotive industry that would require a strong understanding of objects in motion to figure out how to reduce drag to increase fuel efficiency.
2. I miss the freedom to do things with friends and travel while also having time to work and make money.
3. $\sigma = F/A$ This is the equation for calculating stress.

**Comment by QLane22** [comment]
1. A job posting that require skills utilized and taught in this class is biomedical engineering, more specifically biomechanics, as that involves working with prosthetics. With prosthetics, a biomedical engineer basically needs to replicate the limb that the device is replacing so basically they must replicate the motion and degree of flexibility of a human limb into the mechanical limb they are working on to make it as realistic and useful as possible.
2.  I miss the lack of responsibility and the freedom to do what I want to do with a lot more time afforded to me
3. $K = \frac{1}{2} m v^2$ this is the kinetic energy equation which relates an object's mass and velocity when in motion, to the amount of kinetic energy that is being released via the object being put in motion.

**Comment by PacoBvr** [comment]
1. A job posting that require skills learned in this class could be a job in aerospace industry because it needs to understand how the objects move in space and how are the interactions with others objects.
2. I miss the sun and high temperature.
3. $\vec{F} = m \vec{g}$ this is the weight force equation, one of the most common force that each object is subject of.

**Comment by ahm1208** [comment]
1) A job that would require skills from this class would be a civil engineering job that focuses on the building of structures, like buildings and bridges. 
2) I miss the sun and being able to go outside and tan at any point in the day.
3) F=ma is Newton's 2nd Law, describing the relationship between mass and acceleration.

**Comment by AidanJak** [comment]
1. A job that requires skills learned from this class would be a car designer as they need to understand all the different factors that are involved while someone is operating the vehicle.
2. I miss the weather and going to the beach.
3. $F=m*a$ is Newton's Second Law of Motion that describes an object's force through its mass and acceleration.

**Comment by shivpatel2005** [comment]
A job that requires skills that are taught in this class is a job posted by Stoke Space, "Propulsion Engineer." The design of propulsion elements of a fully reusable launch vehicle is a topic that absolutely requires what is discussed in the material of this class.

A thing I miss from the summer is my internship. I felt like I was learning really useful things every day, and I really enjoyed working with the people there.

$F = ma$ is an equation. It is Newton's second law and is used to correlate an object's force required to move with its mass and acceleration provided.

**Comment by DavidKallberg02** [comment]
> 1. Most jobs in mechanical engineering require skills from this class. One example is working in the automotive design industry, which I hope to do one day
> 2. I miss the summer days when the weather wasn't as terrible all the time
> 3. 
>    \begin{theorem} a^2 + b^2 = c^2 \end{theorem}
>    The Pythagorean Theorem is a personal favorite of mine that I like because it can be used to find distance
